apps/trading/components/navbar/navbar.tsx

```diff
@@ -71,6 +71,7 @@ const NavLink = ({ name, path, exact, testId = name }: NavLinkProps) => {
   const linkClasses = classNames('mx-2 py-2 self-end border-b-4', {
     'border-vega-yellow text-white cursor-default': isActive,
     'border-transparent text-neutral-400 hover:text-neutral-300': !isActive,
+    'focus-visible:underline': !isActive,
```

Comment:
See below. I don't think there is much variety - borders are used in some places like block elements like input fields and underline for plain text things like nav/tabs. This is the problem with trying to retro-fit styles to components, rather than designing them with all the states in mind up-front - you'll end up with designs where a single style doesn't work for everything.

Reply:
Have removed for now.